mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -125,6 +125,10 @@
     <!-- Text for the number of tabs in a group in the 'Jump back in' section of the new tab
         %d is a placeholder for the number of sites in the group. This number will always be more than one. -->
     <string name="recent_tabs_search_term_count_2">%d sites</string>
+    <!-- Text for button in synced tab card that opens synced tabs tray -->
+    <string name="recent_tabs_see_all_synced_tabs_button_text">See all synced tabs</string>
+    <!-- Accessibility description for icon for recent synced tab -->
```

Comment:
Shouldn't the comment rather say "Accessibility description for icon for recent synced *device*"? (vs tab)
Or should the corresponding string be updated since it refers to "device" here?
CC @MatthewTighe

Reply:
#24322 fixes it with my suggestion, but happy to update it to yours if you think that makes better sense 🙂